src/scss/pages/carousel.scss

```diff
@@ -23,27 +30,27 @@
  $clr-base: #e0e4fb !default;
  $clr-blue: #465798 !default;
  $clr-darkred: darkred !default;
- 
+
  $transition-time: .3s !default;
  $play-btn-color: $clr-blue !default;
  $play-btn-size: 32px !default;
```

Comment:
looks like a not really proper place for these constants. How did they appear here? It could be not you, I know, but please move them somewhere into a properer place. Tnx